pkg/guardian/redis_conf_store.go

```diff
@@ -485,20 +736,50 @@ func (rs *RedisConfStore) SetLimit(limit Limit) error {
 	pipe.Set(redisLimitCountKey, limitCountStr, 0)
 	pipe.Set(redisLimitDurationKey, limitDurationStr, 0)
 	pipe.Set(redisLimitEnabledKey, limitEnabledStr, 0)
+	pipe.Set(redisUseDeprecatedLimitKey, "true", 0)
 
 	_, err := pipe.Exec()
+	if err != nil {
```

Comment:
Consider leaving `return err` instead of
```
if err != nil { 
     return err
}
     return nil
}
```

Reply:
Whoops, left-over from when I was using a helper method. Fixed.